nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -422,4 +442,27 @@ public void checkTypeParameterNullnessForAssignabilityForConditionalExpression(
       }
     }
   }
+
+  public void compareNullabilityOfGenericParameters(
+      List<Symbol.VarSymbol> formalParams, List<? extends ExpressionTree> actualParams) {
+    if (!config.isJSpecifyMode()) {
+      return;
+    }
+    // Check if the parameters are generics and compare annotations for the actual and formal
+    // parameters
+    for (int i = 0; i < formalParams.size(); i++) {
```

Comment:
We should add a test case where the invoked method has varargs, like these:

https://github.com/uber/NullAway/blob/master/nullaway/src/test/java/com/uber/nullaway/NullAwayVarargsTests.java#L47

We should write something like:

```java
void foo(NullableTypeParam<@Nullable String>... args) { ... }
```
And then _all_ the parameters passed at a call to `foo()` should have `@Nullable String` as their type parameter.

Reply:
Thanks. I will update this.